taskcluster/fenix_taskgraph/__init__.py

```diff
@@ -56,6 +58,11 @@ def get_decision_parameters(graph_config, parameters):
         parameters["target_tasks_method"] = "release"
 
 
+def read_version_file():
+    with open(os.path.join(os.path.dirname(__file__), '..', '..', 'version.txt')) as f:
```

Comment:
👍

I don't remember taskgraph importing any other file in the repo. We may eventually want to extract this logic in a dedicated function (like taskgraph in Gecko does). That said, let's keep it this way for now 😃